apps/schedules/lib/hours_of_operation.ex

```diff
@@ -62,7 +62,8 @@ defmodule Schedules.HoursOfOperation do
         route: route_id,
         date: date,
         direction_id: direction_id,
-        stop_sequence: "first,last",
+        # there appears to be an issue with the new api when using first returning stops that shouldn't be returned
```

Comment:
Have you raised this issue with TRC? If not please file an Asana ticket with them . :)

Reply:
I just tested this on the production api (and dev-blue api).  It doesn't appear to make a difference.
I will revert it

Reply:
I can still reproduce the issue on production and opened a ticket with TRC: https://app.asana.com/0/881264583703207/1203272636155437/f.